src/main/java/uk/gov/ons/ctp/integration/rhsvc/config/InboundEventIntegrationConfig.java

```diff
@@ -211,16 +256,6 @@ private Jackson2JsonMessageConverter jsonMessageConverter(
     return converter;
   }
 
-  @Bean
```

Comment:
thanks for removing these. these were in the XML, and I forgot to remove them later when I noticed they weren't used.

Reply:
No problem.